plugins/hls-qualify-imported-names-plugin/src/Ide/Plugin/QualifyImportedNames.hs

```diff
@@ -111,7 +110,7 @@ data ImportedBy = ImportedBy {
 }
 
 isRangeWithinImportedBy :: Range -> ImportedBy -> Bool
-isRangeWithinImportedBy range (ImportedBy _ srcSpan) = fromMaybe False $ spanContainsRange srcSpan range
+isRangeWithinImportedBy range ImportedBy{importedBySrcSpan} = fromMaybe False $ spanContainsRange importedBySrcSpan range
```

Comment:
Agree that using named field puns could be controversial, I'm just fixing warnings of the type
`Defined but not used: ‘importedBySrcSpan’`

Other ways to make this go away:
- prefix record selectors in data type definition with _
- use the record selectors (e.g. `importedBySrcSpan ib`) in function body

Reply:
I think it is fine, and I don't think we have any policy regarding `NamedFieldPuns`.